tests/TestCase/Http/Cookie/RequestCookiesTest.php

```diff
@@ -31,9 +31,7 @@ class RequestCookiesTest extends TestCase
     public $request;
 
     /**
-     * setup
-     *
-     * @return null
+     * {@inheritDoc}
```

Comment:
@burzum yes it seems so:
https://phpdoc.org/docs/latest/guides/inheritance.html#the-inheritdoc-tag

https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#61-making-inheritance-explicit-using-the-inheritdoc-tag